decidim-core/spec/middleware/decidim/current_organization_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "spec_helper"
+
+module Decidim
+  describe CurrentComponent do
```

Comment:
Again, your previous PR on the subject modified this file instead of copying it, so we lost the specs for the `CurrentOrganization` middleware. Can you copy this file to `current_comnponent.rb` and get back the tests for `CurrentOrganization`?